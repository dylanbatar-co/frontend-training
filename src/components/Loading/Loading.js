import React from 'react';
import SkeletonLoader from 'emerald-ui/lib/SkeletonLoader';

export const Loading = () => {
  return (
    <div aria-hidden className="ml-5 mr-5 mt-2 mb-2">
      <SkeletonLoader height="190px" width="320px" loading highContrast />
      <SkeletonLoader height="40px" width="320px" loading highContrast />
      <SkeletonLoader height="80px" width="320px" loading highContrast />
    </div>
  );
};
