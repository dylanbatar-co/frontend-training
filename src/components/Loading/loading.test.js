import { shallow } from 'enzyme';
import { Loading } from './Loading';

describe('Loading Component Test', () => {
  it('Loading component snapshot ', () => {
    const component = shallow(<Loading />);
    expect(component).toMatchSnapshot();
  });
});
