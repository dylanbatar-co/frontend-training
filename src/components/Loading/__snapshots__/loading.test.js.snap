// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component Test Loading component snapshot  1`] = `
<div
  aria-hidden={true}
  className="ml-5 mr-5 mt-2 mb-2"
>
  <SkeletonLoader
    height="190px"
    highContrast={true}
    loading={true}
    width="320px"
  />
  <SkeletonLoader
    height="40px"
    highContrast={true}
    loading={true}
    width="320px"
  />
  <SkeletonLoader
    height="80px"
    highContrast={true}
    loading={true}
    width="320px"
  />
</div>
`;
